neuralprophet/df_utils.py

```diff
@@ -150,7 +155,7 @@ def init_data_params(
     """
 
     if isinstance(df, list):
-        df_list = df.copy()
+        df_list = copy_list(df)
```

Comment:
Did not consider everyone as a list because of the log info for global normalization or local normalization